app/views/declaration/supervising_customs_office.scala.html

```diff
@@ -14,50 +14,78 @@
+@this(
+    govukLayout: gdsMainTemplate,
+    govukButton: GovukButton,
+    govukRadios: GovukRadios,
+    govukDetails : GovukDetails,
+    govukFieldset: GovukFieldset,
+    errorSummary: errorSummary,
+    sectionHeader: sectionHeader,
+    saveButtons: saveButtons,
+    tariffDetails: tariffDetails,
+    formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[SupervisingCustomsOffice])(implicit request: JourneyRequest[_], messages: Messages)
 
-@tariffBody = { @Html(messages("declaration.type.supervisingCustomsOfficeSummaryText", components.details_content_link()))}
-
-    @main_template(
-        title = Title("declaration.warehouse.supervisingCustomsOffice.title")
-    ) {
+@pageHint = {
+    <span class="govuk-hint">
+        @messages("declaration.warehouse.supervisingCustomsOffice.hint")
+    </span>
+}
 
-        @helper.form(routes.SupervisingCustomsOfficeController.submit(mode), 'autoComplete -> "off") {
-            @helper.CSRF.formField
+@tariff = {
+    @components.details(messages("site.details.summary_text_these")) {
+        @Html(messages("declaration.type.supervisingCustomsOfficeSummaryText", components.details_content_link()))
+    }
+}
 
-            @components.back_link(Navigator.backLink(SupervisingCustomsOffice, mode)(request))
+@autocomplete = {
+    @components.fields.field_accessible_autocomplete(
+        field = form("supervisingCustomsOffice"),
+        label = "",
+        labelClass = None,
+        hintText = None,
+        emptySelectValue = messages("supplementary.packageInformation.typesOfPackages.empty"),
+        items = AutoCompleteItem.fromSupervisingCustomsOffice(CustomsOffices.all)
+    )
+}
 
-            @components.error_summary(form.errors)
+@govukLayout(
+    title = Title("declaration.warehouse.supervisingCustomsOffice.title"),
+    backButton = Some(BackButton(messages("site.back"), Navigator.backLink(SupervisingCustomsOffice, mode)))){
 
-            @components.heading(
-                sectionHeader = messages("declaration.warehouse.supervisingCustomsOffice.sectionHeader"),
-                title = messages("declaration.warehouse.supervisingCustomsOffice.title"),
-                hint = messages("declaration.warehouse.supervisingCustomsOffice.hint")
-            )
+    @formHelper(action = controllers.declaration.routes.SupervisingCustomsOfficeController.submit(mode), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
-            @components.fields.field_autocomplete(
-                field = form("supervisingCustomsOffice"),
-                label = messages(""),
-                labelClass = Some("form-label-bold"),
-                hintText = None,
-                emptySelectValue = messages("supplementary.packageInformation.typesOfPackages.empty"),
-                items = AutoCompleteItem.fromSupervisingCustomsOffice(CustomsOffices.all))
+        @sectionHeader(messages("declaration.warehouse.supervisingCustomsOffice.sectionHeader"))
```

Comment:
I think that the secitonHeader can be moved to the autocomplete as label with custom class, but not sure about it.

Reply:
Hum... I am not sure about that, as this is the way we tend to do things. Admittedly there are two labels missing as part of the autocomplete component: one for the select tag and another for the input text tag.